doc/nrf/ug_nrf5340.rst

```diff
@@ -36,7 +36,10 @@ Application core
 
 The application core is a full-featured Arm Cortex-M33 processor including DSP instructions and FPU.
 
-In the current implementation of the nRF5340 samples, this core is used to run the high-level radio stack (the host part of the Bluetooth Low Energy stack) and the application.
+Currently, in Bluetooth LE context, the following solutions are available for the application core:
+* High-level radio stack (the host part of the Bluetooth Low Energy stack) and application logic,
+* A sample with pure application logic (with whole radio connectivity implementation moved to the network core)
```

Comment:
I guess this is a situation when we flash whole Bluetooth sample on the network core. I don't understand this sentence a bit maybe It could be more describing

Reply:
I will re-write, or remove at all, depending on what is decided with https://github.com/NordicPlayground/fw-nrfconnect-nrf/pull/1819, see https://github.com/NordicPlayground/fw-nrfconnect-nrf/pull/1819#issuecomment-580823839

Reply:
Removed, since #1819 was rejected for NCS 1.2.0.